packages/tools/pluggable-widgets-tools/src/typings-generator/__tests__/inputs/containment.ts

```diff
@@ -23,7 +23,7 @@ export const containmentInput = `<?xml version="1.0" encoding="utf-8"?>
             </property>
         </propertyGroup>
         <propertyGroup caption="System Properties">
-            <systemProperty key="Label"></systemProperty>
+            <!-- no "Label" system property -->
```

Comment:
What is the reason for not having a label system property?

Reply:
If the widget does not have the system property `Label`, the widget generator should not generate the `id` prop and I wanted to make sure that this was covered in at least some of the tests, rather than adding `id` in the output everywhere. I picked "containment" for this because it's a kind of widget that is less likely to be labelled in practice.